xcube_cds/datasets/reanalysis_era5.py

```diff
@@ -23,10 +23,12 @@
 import json
 import os
 import pathlib
-from typing import List, Optional, Dict, Tuple
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Tuple
```

Comment:
Its absolutely fine, but in xcube code we single component imports only for our own components.

Reply:
If it's fine, I'll stick with this "single import for everything" convention for the CDS Store.